hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2966,6 +2967,40 @@ public String getRpcPort() {
   }
 
   @Override
+
+  public String getOmRatisRoles() {
+    List<ServiceInfo> serviceList = null;
+    StringBuilder sb = new StringBuilder();
+    try {
+      serviceList = getServiceList();
+    } catch (IOException e) {
+      return "";
+    }
+    for (ServiceInfo info : serviceList) {
+      if (info.getOmRoleInfo() != null) {
+        sb.append(
+            String.format(
+                "{ HostName: %s, Ratis Port: %s, Node-Id: %s, Role: %s } ",
+                info.getHostname(),
+                info.getPort(ServicePort.Type.RATIS),
+                info.getOmRoleInfo().getNodeId(),
+                info.getOmRoleInfo().getServerRole()
+            ));
+      }
+    }
+    return sb.toString();
+  }
+
+  @Override
+  public String getCurrentHost() {
```

Comment:
I don’t think we need to add the hostname, Without knowing the hostname you wouldn’t be able to access UI itself.

Reply:
Sure Ill remove it 
Thanks !